frontend/src/components/stats/GasUsedByDate.tsx

```diff
@@ -1,21 +1,27 @@
 import React, { useState, useEffect } from "react";
+import { Tabs, Tab } from "react-bootstrap";
 import ReactEcharts from "echarts-for-react";
 import echarts from "echarts";
 
 import StatsApi, {
   TeragasUsedByDate,
 } from "../../libraries/explorer-wamp/stats";
+import { cumulativeSumArray } from "../../libraries/formatting";
 
-export default () => {
+import { Props } from "./TransactionsByDate";
+
+export default ({ chartStyle }: Props) => {
   const [teragasUsedByDate, setTeragasUsedByDate] = useState(Array());
   const [date, setDate] = useState(Array());
+  const [total, setTotal] = useState(Array());
```

Comment:
@icerove "total" is a too broad term. We spend too much time on naming, but that is only to have the code-base sane for future maintenance. Please, be mindful of the naming. You had done a good job with `teragasUsedByDate`, following the style, I suggest `cumulativeTeragasUsedByDate` instead of `total`.